apps/site-landing/src/containers/HelpDocs/FaqMenu.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Link } from 'react-router-dom';
+import styles from "./styles.module.less";
+import { List, Rail } from "semantic-ui-react";
+import { GreaterThanMobileSegment, MobileSegment } from "@thecointech/shared/components/ResponsiveTool";
+
+type Props = {
+    categories: any
+  }
+  
+function displayEntry(index: number, name: string ){
+    return <List.Item key={index}><Link to={"/faq/theme-"+((name)?.split("-"))[0].replace(/ /g, '')}>{((name.split("-"))[1])}</Link>
+                { /* <List> */ }
```

Comment:
I thought it could be better to have the list of questions under the FAQ categories in the menu but I wanted to see that with you first.

Reply:
That sounds good, but I'm not sure we would want to do it until we had enough questions, And then we'd want to take it away again when we had too many.

I'm fairly ambivalent.  Wouldn't take it out, wouldn't put it in.